crates/bitwarden-core/Cargo.toml

```diff
@@ -54,6 +54,9 @@ validator = { version = "0.18.1", features = ["derive"] }
 zeroize = { version = ">=1.7.0, <2.0", features = ["derive", "aarch64"] }
 zxcvbn = { version = ">=3.0.1, <4.0", optional = true }
 
+[target.'cfg(debug_assertions)'.dependencies]
+tokio = { version = "1.36.0", features = ["rt", "macros", "time"] }
```

Comment:
I barely know what I did here, I need the time feature but only in debug, is this the right way to go?

Reply:
It is, but I can't say I'm a fan of different behaviors on debug and release builds.

Reply:
This feels like the exact thing a debug build is for, it helps you write tests to make sure you've built the language wrapper correctly. That said I was able to move it into the `bitwarden-json` crate instead of `core`.